deb package builds broken

https://github.com/systemd/systemd/pull/42118 broke all deb package builds:

```
[  501s] make[1]: Leaving directory '/usr/src/packages/BUILD'
[  501s]    dh_dwz -a
[  502s] dwz: dwz: debian/libudev1/usr/lib/x86_64-linux-gnu/libudev.so.1.7.14: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] debian/libnss-myhostname/usr/lib/x86_64-linux-gnu/libnss_myhostname.so.2: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dh_dwz: error: dwz -- debian/libudev1/usr/lib/x86_64-linux-gnu/libudev.so.1.7.14 returned exit code 1
[  502s] dh_dwz: error: dwz -- debian/libnss-myhostname/usr/lib/x86_64-linux-gnu/libnss_myhostname.so.2 returned exit code 1
[  502s] dwz: debian/systemd-boot/usr/lib/systemd/system-generators/systemd-bless-boot-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd-boot/usr/lib/systemd/systemd-bless-boot: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: Too few files for multifile optimization
[  502s] dh_dwz: error: dwz -mdebian/systemd-boot/usr/lib/debug/.dwz/x86_64-linux-gnu/systemd-boot.debug -M/usr/lib/debug/.dwz/x86_64-linux-gnu/systemd-boot.debug -- debian/systemd-boot/usr/lib/systemd/system-generators/systemd-bless-boot-generator debian/systemd-boot/usr/lib/systemd/systemd-bless-boot returned exit code 1
[  502s] dwz: debian/systemd/usr/bin/busctl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/hostnamectl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/journalctl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/kernel-install: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/localectl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/loginctl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/networkctl: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemctl: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-ac-power: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-analyze: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-ask-password: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-cat: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-cgls: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-cgtop: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-creds: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-delta: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-detect-virt: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-escape: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-firstboot: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-id128: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-inhibit: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-machine-id-setup: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-mount: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-mute-console: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-notify: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-path: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-pty-forward: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-run: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-socket-activate: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-stdio-bridge: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-sysext: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-sysinstall: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-sysusers: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-tmpfiles: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-tty-ask-password-agent: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/systemd-vpick: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/timedatectl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/bin/varlinkctl: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-debug-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-factory-reset-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-fstab-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-getty-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-gpt-auto-generator: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-hibernate-resume-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-imds-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-run-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-ssh-generator: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-system-update-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/system-generators/systemd-tpm2-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-backlight: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-battery-check: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-binfmt: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-boot-check-no-failures: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-bsod: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-executor: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-factory-reset: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-fsck: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-growfs: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-hibernate-resume: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-hostnamed: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-imds: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-imdsd: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-journald: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-localed: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-logind: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-makefs: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-measure: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-modules-load: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-network-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-networkd: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-networkd-wait-online: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-pcrextend: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-pcrlock: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-pstore: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-quotacheck: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-random-seed: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-remount-fs: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-reply-password: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-report: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-report-basic: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-report-cgroup: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-rfkill: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-shutdown: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-sleep: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-socket-proxyd: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-ssh-issue: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-ssh-proxy: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-storagetm: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-sulogin-shell: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-sysctl: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-timedated: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-tpm2-clear: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-tpm2-setup: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-tpm2-swtpm: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-update-done: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-user-runtime-dir: Found compressed .debug_aranges section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-user-sessions: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-validatefs: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-volatile-root: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/systemd-xdg-autostart-condition: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/user-environment-generators/30-systemd-environment-d-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: debian/systemd/usr/lib/systemd/user-generators/systemd-xdg-autostart-generator: Found compressed .debug_info section, not attempting dwz compression
[  502s] dwz: Too few files for multifile optimization
[  502s] dh_dwz: error: dwz -mdebian/systemd/usr/lib/debug/.dwz/x86_64-linux-gnu/systemd.debug -M/usr/lib/debug/.dwz/x86_64-linux-gnu/systemd.debug -- debian/systemd/usr/bin/busctl debian/systemd/usr/bin/hostnamectl debian/systemd/usr/bin/journalctl debian/systemd/usr/bin/kernel-install debian/systemd/usr/bin/localectl debian/systemd/usr/bin/loginctl debian/systemd/usr/bin/networkctl debian/systemd/usr/bin/systemctl debian/systemd/usr/bin/systemd-ac-power debian/systemd/usr/bin/systemd-analyze debian/systemd/usr/bin/systemd-ask-password debian/systemd/usr/bin/systemd-cat debian/systemd/usr/bin/systemd-cgls debian/systemd/usr/bin/systemd-cgtop debian/systemd/usr/bin/systemd-creds debian/systemd/usr/bin/systemd-delta debian/systemd/usr/bin/systemd-detect-virt debian/systemd/usr/bin/systemd-escape debian/systemd/usr/bin/systemd-firstboot debian/systemd/usr/bin/systemd-id128 debian/systemd/usr/bin/systemd-inhibit debian/systemd/usr/bin/systemd-machine-id-setup debian/systemd/usr/bin/systemd-mount debian/systemd/usr/bin/systemd-mute-console debian/systemd/usr/bin/systemd-notify debian/systemd/usr/bin/systemd-path debian/systemd/usr/bin/systemd-pty-forward debian/systemd/usr/bin/systemd-run debian/systemd/usr/bin/systemd-socket-activate debian/systemd/usr/bin/systemd-stdio-bridge debian/systemd/usr/bin/systemd-sysext debian/systemd/usr/bin/systemd-sysinstall debian/systemd/usr/bin/systemd-sysusers debian/systemd/usr/bin/systemd-tmpfiles debian/systemd/usr/bin/systemd-tty-ask-password-agent debian/systemd/usr/bin/systemd-vpick debian/systemd/usr/bin/timedatectl debian/systemd/usr/bin/varlinkctl debian/systemd/usr/lib/systemd/system-generators/systemd-debug-generator debian/systemd/usr/lib/systemd/system-generators/systemd-factory-reset-generator debian/systemd/usr/lib/systemd/system-generators/systemd-fstab-generator debian/systemd/usr/lib/systemd/system-generators/systemd-getty-generator debian/systemd/usr/lib/systemd/system-generators/systemd-gpt-auto-generator debian/systemd/usr/lib/systemd/system-generators/systemd-hibernate-resume-generator debian/systemd/usr/lib/systemd/system-generators/systemd-imds-generator debian/systemd/usr/lib/systemd/system-generators/systemd-run-generator debian/systemd/usr/lib/systemd/system-generators/systemd-ssh-generator debian/systemd/usr/lib/systemd/system-generators/systemd-system-update-generator debian/systemd/usr/lib/systemd/system-generators/systemd-tpm2-generator debian/systemd/usr/lib/systemd/systemd debian/systemd/usr/lib/systemd/systemd-backlight debian/systemd/usr/lib/systemd/systemd-battery-check debian/systemd/usr/lib/systemd/systemd-binfmt debian/systemd/usr/lib/systemd/systemd-boot-check-no-failures debian/systemd/usr/lib/systemd/systemd-bsod debian/systemd/usr/lib/systemd/systemd-executor debian/systemd/usr/lib/systemd/systemd-factory-reset debian/systemd/usr/lib/systemd/systemd-fsck debian/systemd/usr/lib/systemd/systemd-growfs debian/systemd/usr/lib/systemd/systemd-hibernate-resume debian/systemd/usr/lib/systemd/systemd-hostnamed debian/systemd/usr/lib/systemd/systemd-imds debian/systemd/usr/lib/systemd/systemd-imdsd debian/systemd/usr/lib/systemd/systemd-journald debian/systemd/usr/lib/systemd/systemd-localed debian/systemd/usr/lib/systemd/systemd-logind debian/systemd/usr/lib/systemd/systemd-makefs debian/systemd/usr/lib/systemd/systemd-measure debian/systemd/usr/lib/systemd/systemd-modules-load debian/systemd/usr/lib/systemd/systemd-network-generator debian/systemd/usr/lib/systemd/systemd-networkd debian/systemd/usr/lib/systemd/systemd-networkd-wait-online debian/systemd/usr/lib/systemd/systemd-pcrextend debian/systemd/usr/lib/systemd/systemd-pcrlock debian/systemd/usr/lib/systemd/systemd-pstore debian/systemd/usr/lib/systemd/systemd-quotacheck debian/systemd/usr/lib/systemd/systemd-random-seed debian/systemd/usr/lib/systemd/systemd-remount-fs debian/systemd/usr/lib/systemd/systemd-reply-password debian/systemd/usr/lib/systemd/systemd-report debian/systemd/usr/lib/systemd/systemd-report-basic debian/systemd/usr/lib/systemd/systemd-report-cgroup debian/systemd/usr/lib/systemd/systemd-rfkill debian/systemd/usr/lib/systemd/systemd-shutdown debian/systemd/usr/lib/systemd/systemd-sleep debian/systemd/usr/lib/systemd/systemd-socket-proxyd debian/systemd/usr/lib/systemd/systemd-ssh-issue debian/systemd/usr/lib/systemd/systemd-ssh-proxy debian/systemd/usr/lib/systemd/systemd-storagetm debian/systemd/usr/lib/systemd/systemd-sulogin-shell debian/systemd/usr/lib/systemd/systemd-sysctl debian/systemd/usr/lib/systemd/systemd-timedated debian/systemd/usr/lib/systemd/systemd-tpm2-clear debian/systemd/usr/lib/systemd/systemd-tpm2-setup debian/systemd/usr/lib/systemd/systemd-tpm2-swtpm debian/systemd/usr/lib/systemd/systemd-update-done debian/systemd/usr/lib/systemd/systemd-user-runtime-dir debian/systemd/usr/lib/systemd/systemd-user-sessions debian/systemd/usr/lib/systemd/systemd-validatefs debian/systemd/usr/lib/systemd/systemd-volatile-root debian/systemd/usr/lib/systemd/systemd-xdg-autostart-condition debian/systemd/usr/lib/systemd/user-environment-generators/30-systemd-environment-d-generator debian/systemd/usr/lib/systemd/user-generators/systemd-xdg-autostart-generator returned exit code 1
[  502s] dh_dwz: error: Aborting due to earlier error
[  502s] make: *** [debian/rules:337: binary] Error 25
[  502s] dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
[  502s] ### VM INTERACTION START ###
[  502s] [  492.458060][    T1] sysrq: Power Off
[  502s] [  492.458937][    T7] reboot: Power down
[  502s] ### VM INTERACTION END ###
```

https://build.opensuse.org/package/live_build_log/system:systemd/systemd/Debian_13/x86_64

--- a/meson.build
+++ b/meson.build
@@ -501,12 +501,6 @@ possible_cc_flags = [
 
 if get_option('mode') == 'developer'
         possible_cc_flags += '-fno-omit-frame-pointer'
-        # Let -Wl,--gc-sections drop unused functions/data instead of whole
-        # .o files; biggest win on statically-linked NSS/PAM modules.
-        possible_cc_flags += ['-ffunction-sections', '-fdata-sections']
-        # Compress and dedupe DWARF; transparent to GDB / readelf.
-        possible_cc_flags += ['-gz=zstd', '-fdebug-types-section']
-        possible_link_flags += '-Wl,--compress-debug-sections=zstd'
 endif
 
 add_project_arguments(

--- a/mkosi/mkosi.images/build/mkosi.conf.d/debian-ubuntu/mkosi.build.chroot
+++ b/mkosi/mkosi.images/build/mkosi.conf.d/debian-ubuntu/mkosi.build.chroot
@@ -40,12 +40,6 @@ if [[ -d "$SRCDIR/debian/patches" ]]; then
     mount --bind /tmp/patches "$SRCDIR/debian/patches"
 fi
 
-# mode=developer enables -Wl,--compress-debug-sections=zstd in meson.build, which dwz cannot process.
-cp "$SRCDIR/debian/rules" /tmp/rules
-chmod +x /tmp/rules
-printf '\noverride_dh_dwz:\n' >>/tmp/rules
-mount --bind /tmp/rules "$SRCDIR/debian/rules"
-
 # While the build directory can be specified through DH_OPTIONS, the default one is hardcoded everywhere so
 # we have to use that. Because it is architecture dependent, we query it using dpkg-architecture first.
 DEB_HOST_GNU_TYPE="$(dpkg-architecture --query DEB_HOST_GNU_TYPE)"

